src/translator/service.h

```diff
@@ -112,16 +108,14 @@ class BlockingService {
 class AsyncService {
  public:
   struct Config {
-    size_t numWorkers{1};      ///< How many worker translation threads to spawn.
-    bool cacheEnabled{false};  ///< Whether to enable cache or not.
-    size_t cacheSize{2000};    ///< Size in History items to be stored in the cache. Loosely corresponds to sentences to
-                               /// cache in the real world.
-    Logger::Config logger;     // Configurations for logging
+    size_t numWorkers{1};    ///< How many worker translation threads to spawn.
+    size_t cacheSize{2000};  ///< Size in History items to be stored in the cache. Loosely corresponds to sentences to
```

Comment:
Should this now default to 0? Previously cache was disabled by default. 

(Same for BlockingService btw)